modules/SitecoreImageBuilder/1.0.0/Public/Invoke-Build.ps1

```diff
@@ -197,10 +201,17 @@ function Invoke-Build
             # Build image
             $buildOptions = New-Object System.Collections.Generic.List[System.Object]
 
-            if ($osType -eq "windows")
-            {
+            if ($osType -ieq "windows" -and $IsolationModeBehaviour -ieq "ForceHyperV") {
+                # --isolation 'hyperv' | makes sense on windows host only?
                 $buildOptions.Add("--isolation 'hyperv'")
             }
+            elseif ($IsolationModeBehaviour -ieq "ForceProcess") {
+                # --isolation 'process' | makes sense on all operating systems
+                $buildOptions.Add("--isolation 'process'")
```

Comment:
This needs to be in the ostype==windows statement also as it only applies to windows. On linux you get:
```
docker: Error response from daemon: Invalid isolation: "process" - linux only supports 'default'.
```

Reply:
Fixed